presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroupManager.java

```diff
@@ -200,7 +200,6 @@ public void start()
                 }
                 catch (Throwable t) {
                     log.error(t, "Error while executing refreshAndStartQueries");
-                    throw t;
```

Comment:
It is better to not to schedule rather than to die midway everytime. Detecting and fixing no scheduling is easier than partial scheduling.

Reply:
Makes sense, let me revert it then